.github/workflows/main.yml

```
@@ -18,14 +18,26 @@ jobs:
     strategy:
       matrix:
         use_cpm: ['ON', 'OFF']
+        separate_compilation: ['ON', 'OFF']
+        dyn_link: ['ON', 'OFF']
         no_deprecated: ['ON', 'OFF']
         build_type: ['Release']
         os: [ubuntu-latest, macos-latest, windows-latest]
-        cmake_version: [ latestrc ]
+        cmake_version: [latestrc]
```

```suggestion
        cmake_version: [ latestrc ]
```

No need to change this line